src/Module/Api/Mastodon/Search.php

```diff
@@ -137,6 +137,9 @@ private static function searchAccounts(int $uid, string $q, bool $resolve, int $
 	private static function searchStatuses(int $uid, string $q, string $account_id, int $max_id, int $min_id, int $limit, int $offset)
 	{
 		if (Network::isValidHttpUrl($q)) {
+			if ($offset != 0) {
+				return [];
```

Comment:
The search should only return an empty result set if the `$offset` parameter is greater than the total number of matches.